features/comments_and_kudos/comments_adminposts.feature

```diff
@@ -35,3 +35,74 @@ Feature: Commenting on admin posts
       And all emails have been delivered
     When I edit a comment
     Then 1 email should be delivered to "narcis"
+
+  Scenario: Admin post with comments disabled
+    Given I have posted an admin post
+      And I am logged in as a "communications" admin
+    When I go to the admin-posts page
+      And I follow "Edit"
+    Then I should see "Who can comment on this"
+      And I should see "No one can comment"
+    When I choose "No one can comment"
+      And I press "Post"
+    Then I should see "successfully updated"
```

Comment:
We're testing manually right now and noticed an issue where, if you go back to the admin post's edit page, the right option isn't always selected. It might just be related to some server work, but even if it is, could you expand this test to cover that behavior?

e.g. 
```
When I follow "Edit Post"
Then "No one can comment" should be checked
```

(I _think_ the "should be checked" step works with radio buttons in addition to checkboxes, but I'm not positive.)

Reply:
I wasn't able to reproduce, but I can add these steps for all 3 of the tests around this.

`Then the "No one can comment" checkbox should be checked`

Reply:
It can go onto the existing one -- thank you!